antareslauncher/use_cases/retrieve/final_zip_extractor.py

```diff
@@ -20,7 +20,7 @@ def extract_final_zip(self, study: StudyDTO) -> None:
         Args:
             study: The current study
         """
-        if not study.finished or study.with_error or not study.local_final_zipfile_path or study.final_zip_extracted:
```

Comment:
We should probably keep the conditions on the path and on the "extracted" boolean to ensure we don't perform the extraction twice